pkg/resource/deploy/step.go

```diff
@@ -341,7 +341,7 @@ func (s *DeleteStep) Apply(preview bool) (resource.Status, StepCompleteFunc, err
 			"as it is currently marked for protection. To unprotect the resource, "+
 			"either remove the `protect` flag from the resource in your Pulumi"+
 			"program and run `pulumi up` or use the command:\n"+
-			"`pulumi state unprotect %s`", s.old.URN, s.old.URN)
+			"`pulumi state unprotect `%s``", s.old.URN, s.old.URN)
```

Comment:
```suggestion
			"`pulumi state unprotect '%s'`", s.old.URN, s.old.URN)
```
I agree with @svangordon-fruit. We should use `'`. Backtick doesn't work on bash or zsh or fish, off the top of my head.